docs/src/app/components/pages/components/Chip/ExampleArray.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Chip from 'material-ui/Chip';
+
+const styles = {
+  chip: {
+    margin: 4,
+  },
+  wrapper: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+};
+
+let chipData = [
+  {key: 0, label: 'Angular'},
+  {key: 1, label: 'JQuery'},
+  {key: 2, label: 'Polymer'},
+  {key: 3, label: 'ReactJS'},
+];
+
+/**
+ * An example of rendering multiple Chips from an array of values.
+ */
+export default class ChipExampleSimple extends React.Component {
+
+  constructor(props) {
```

Comment:
Here here the adoption rate compared to [es2015](https://www.npmjs.com/package/babel-preset-es2015):
 - [babel-preset-stage-3](https://www.npmjs.com/package/babel-preset-stage-3): 43%
 - [babel-preset-stage-2](https://www.npmjs.com/package/babel-preset-stage-2): 42%
 - [babel-preset-stage-1](https://www.npmjs.com/package/babel-preset-stage-1): 32%
 - [babel-preset-stage-0](https://www.npmjs.com/package/babel-preset-stage-0): 26%